element_array_ephys/ephys_acute.py

```diff
@@ -694,6 +918,22 @@ def get_spikeglx_meta_filepath(ephys_recording_key):
     return spikeglx_meta_filepath
 
 
+def get_openephys_probe_data(ephys_recording_key):
+    inserted_probe_serial_number = (ProbeInsertion * probe.Probe
+                                    & ephys_recording_key).fetch1('probe')
+    session_dir = find_full_path(get_ephys_root_data_dir(),
+                              get_session_directory(ephys_recording_key))
+    loaded_oe = openephys.OpenEphys(session_dir)
+    probe_data = loaded_oe.probes[inserted_probe_serial_number]
+
+    # explicitly garbage collect "loaded_oe"
+    # as these may have large memory footprint and may not be cleared fast enough
+    del loaded_oe
+    gc.collect()
```

Comment:
@dimitri-yatsenko Your [comment](https://github.com/datajoint/element-array-ephys/pull/71#discussion_r972332982) on PR #71 is regarding an add from PR #54. PR71 assumes 54 will be merged first, which is why it is marked as draft.

I'm not about the details on garbage collection. Comments here indicate that `loaded_oe` has significant memory demands